pom.xml

```diff
@@ -30,6 +30,22 @@
                     <groupId>com.google.cloud.tools</groupId>
                     <artifactId>jib-maven-plugin</artifactId>
                     <version>2.7.0</version>
+                    <configuration>
+                        <container>
+                            <jvmFlags>
+                                <jvmFlag>-Dlogback.configurationFile=/root/conf/logback.xml</jvmFlag>
+                                <jvmFlag>-Dworking.directory=/root/</jvmFlag>
+                                <!-- Prevents Logjam (CVE-2015-4000) -->
+                                <jvmFlag>-Djdk.tls.ephemeralDHKeySize=2048</jvmFlag>
+                                <jvmFlag>--add-opens=java.base/java.lang=ALL-UNNAMED</jvmFlag>
+                                <jvmFlag>--add-opens=java.base/java.math=ALL-UNNAMED</jvmFlag>
+                                <jvmFlag>--add-opens=java.base/java.util=ALL-UNNAMED</jvmFlag>
+                                <jvmFlag>--add-opens=java.base/java.util.concurrent=ALL-UNNAMED</jvmFlag>
+                                <jvmFlag>--add-opens=java.base/java.net=ALL-UNNAMED</jvmFlag>
+                                <jvmFlag>--add-opens=java.base/java.text=ALL-UNNAMED</jvmFlag>
```

Comment:
Just curious, why we need all those jvm flags?

Reply:
As I research, We use maven compile on java 11, then we try to run it on java 18 (here is eclipse java 18-jre-forcal).
We got the error `java.lang.reflect.InaccessibleObjectException` when run deployment-test. 
Some topics on StackOverflow guide this solution.

Reply:
Maybe we don't need them when we use maven compile on java 17. 
Let me make another pr